wodles/aws/constants.py

```diff
@@ -0,0 +1,156 @@
+THROTTLING_EXCEPTION_ERROR_NAME = "ThrottlingException"
+
+# ERROR CODES
+UNKNOWN_ERROR_CODE = 1
+INVALID_CREDENTIALS_ERROR_CODE = 3
+METADATA_ERROR_CODE = 5
+UNABLE_TO_CREATE_DB = 6
+UNEXPECTED_ERROR_WORKING_WITH_S3 = 7
+DECOMPRESS_FILE_ERROR_CODE = 8
+PARSE_FILE_ERROR_CODE = 9
+UNABLE_TO_CONNECT_SOCKET_ERROR_CODE = 11
+INVALID_TYPE_ERROR_CODE = 12
+SENDING_MESSAGE_SOCKET_ERROR_CODE = 13
+EMPTY_BUCKET_ERROR_CODE = 14
+INVALID_ENDPOINT_ERROR_CODE = 15
+THROTTLING_ERROR_CODE = 16
+INVALID_KEY_FORMAT_ERROR_CODE = 17
+INVALID_PREFIX_ERROR_CODE = 18
+INVALID_REQUEST_TIME_ERROR_CODE = 19
+UNABLE_TO_FETCH_DELETE_FROM_QUEUE = 21
+INVALID_REGION_ERROR_CODE = 22
+
+# ERROR MESSAGES
+UNKNOWN_ERROR_MESSAGE = "Unexpected error: '{error}'."
+INVALID_CREDENTIALS_ERROR_MESSAGE = "Invalid credentials to access S3 Bucket"
+INVALID_REQUEST_TIME_ERROR_MESSAGE = "The server datetime and datetime of the AWS environment differ"
+THROTTLING_EXCEPTION_ERROR_MESSAGE = "The '{name}' request was denied due to request throttling. " \
+                                     "If the problem persists check the following link to learn how to use " \
+                                     f"the Retry configuration to avoid it: '{RETRY_CONFIGURATION_URL}'"
+
+# TESTS VALUES
+TEST_HARDCODED_WAZUH_VERSION = "4.5.0"
+TEST_TABLE_NAME = "cloudtrail"
+TEST_SERVICE_NAME = "s3"
+TEST_AWS_PROFILE = "test_aws_profile"
+TEST_IAM_ROLE_ARN = "arn:aws:iam::123455678912:role/Role"
+TEST_IAM_ROLE_DURATION = '3600'
+TEST_ACCOUNT_ID = "123456789123"
+TEST_ACCOUNT_ALIAS = "test_account_alias"
+TEST_ORGANIZATION_ID = "test_organization_id"
+TEST_TOKEN = 'test_token'
+TEST_CREATION_DATE = "2022-01-01"
+TEST_BUCKET = "test-bucket"
+TEST_SERVICE = "test-service"
+TEST_SQS_NAME = "test-sqs"
+TEST_PREFIX = "test_prefix"
+TEST_SUFFIX = "test_suffix"
+TEST_REGION = "us-east-1"
+TEST_DISCARD_FIELD = "test_field"
+TEST_DISCARD_REGEX = "test_regex"
+TEST_ONLY_LOGS_AFTER = "20220101"
+TEST_ONLY_LOGS_AFTER_WITH_FORMAT = "2022-01-01 00:00:00.0"
+TEST_LOG_KEY = "123456789_CloudTrail-us-east-1_20190401T00015Z_aaaa.json.gz"
+TEST_FULL_PREFIX = "base/account_id/service/region/"
+TEST_EXTERNAL_ID = "external-id-Sec-Lake"
+TEST_MESSAGE = "test_message"
+
+TEST_SERVICE_ENDPOINT = 'test_service_endpoint'
+TEST_STS_ENDPOINT = "test_sts_endpoint"
+
+TEST_LOG_FULL_PATH_CLOUDTRAIL_1 = 'AWSLogs/123456789/CloudTrail/us-east-1/2019/04/01/123456789_CloudTrail-us-east-1_' \
+                                  '20190401T0030Z_aaaa.json.gz'
+TEST_LOG_FULL_PATH_CLOUDTRAIL_2 = 'AWSLogs/123456789/CloudTrail/us-east-1/2019/04/01/123456789_CloudTrail-us-east-1_' \
+                                  '20190401T00015Z_aaaa.json.gz'
+TEST_LOG_FULL_PATH_CUSTOM_1 = 'custom/2019/04/15/07/firehose_custom-1-2019-04-15-09-16-03.zip'
+TEST_LOG_FULL_PATH_CUSTOM_2 = 'custom/2019/04/15/07/firehose_custom-1-2019-04-15-13-19-03.zip'
+TEST_LOG_FULL_PATH_CONFIG_1 = 'AWSLogs/123456789/Config/us-east-1/2019/4/15/ConfigHistory/123456789_Config_us-east-1_' \
```

Comment:
I would define these constans inside the `tests` repository. Someone importing `constants` in production code shouldn't be shown these options.

Reply:
Fixed in commit 94d25014f05f5498e403b62901aba5c5e55c846f.